### ISSUE TITLE
Problem with TouchUI

### ISSUE THREAD

**Issue by rudirud** [Birkbjo/OctoPrint-Themeify#42]
Hey, first: One of my favorite Plugins for Octoprint. Nice work !! But since I installed TouchUI (https://plugins.octoprint.org/plugins/touchui/) my touch screen looks strange. I'm not a programmer, but i would be happy if touchid would not (or only on some special options) use your themeify settings. I dont know if this is even possible... Best Regards David

**Comment by Birkbjo** [comment]
It should already disable the theming, are you taking about the customisable options? 
I will look into disabling those when touchUI is enabling. Seems reasonable, as that has its own customisation.

**Comment by rudirud** [comment]
Yes, i made some Custom settings in wide. So he destroys my Touch Ui Frontend. Thanks for your fast respond! Best Regards from Germany

**Comment by BennyBoyW** [comment]
Was there a resolution to this issue as my TouchUI has been broken by Themeify too ?

**Comment by pandel** [comment]
First things first! Many thanks for your great plugin. It makes Octoprint look so nice! And I love the customization possibilities :-) ...

But I'd also like to vote for the possibility to disable all customizations when using TouchUI.

Keep up the good work!

**Comment by pandel** [comment]
FWIW, what I learned regarding customizations, one can easily add the ".themeify" selector in front of every other customized css selector to disable it while using TouchUI.

**Comment by BennyBoyW** [comment]
@pandel - can you give more detail to a CSS noob on how to do that ?

**Comment by pandel** [comment]
Let's say, you want to customize the width of your Octoprint main view. Then you have to add too customizations (my values are examples, choose your own) in your Themeify plugin configuration:

```
Selector                  | CSS-Rule     | Value 
.themeify .container        width          1200px;
.themeify .span8            width          1200px;
```

**Comment by BennyBoyW** [comment]
Thank you so much - very helpful indeed

**Comment by Birkbjo** [comment]
Great that you found a workaround! I will look into disabling it all together when using it with TouchUI though, as that has its own customizations!

**Comment by BennyBoyW** [comment]
That's great to hear. Themeify and TouchUI are the plugins I most value. TouchUI just makes everything look brilliant. Thanks for creating it :)
